src/accelerate/commands/launch.py

```diff
@@ -625,6 +625,21 @@ def launch_command_parser(subparsers=None):
         ),
     )
 
+    # MPI arguments
+    mpirun_args = parser.add_argument_group("MPI Arguments", "Arguments related to mpirun for Multi-CPU")
+    mpirun_args.add_argument(
+        "--mpirun_hostfile",
+        type=str,
+        default=None,
+        help="Location for a hostfile for using Accelerate to launch a multi-CPU training job with mpirun.",
```

Comment:
IIUC, depending on the MPI implementation being used, the arguments can have different names. Would it be worth documenting that?